### ISSUE TITLE
Plugin idea : csv import to generate accounts ?

### ISSUE THREAD

**Issue by bristow** [Chocobozzz/PeerTube#2484]
While waiting for Ldap/Cas authentication plugin, maybe we could consider a plugin to add mass accounts ?
Currently, there are only 7 fields to create an account :

1. username
2. email
3. password (could be generated automatically)
4. role
5. video quota
6. daily videoquota
7. box: bypass video auto blacklist

What do you think? How about you @Chocobozzz ?

**Comment by JohnXLivingston** [comment]
You can easily develop a script that create these users via API call. I think that a plugin is overkill, since there will probably only be one or two users that will import these files.

**Comment by bristow** [comment]
Indeed, it would be for administrators but i'm not a developper and it could be very useful in my case :)

**Comment by JohnXLivingston** [comment]
Yes, it can be useful. 
For the record, I make a pull request that was just merge: https://github.com/Chocobozzz/PeerTube/pull/2479
The password is no more required. If left blank, an email will be sent to the user so he can set his password.

**Comment by bristow** [comment]
Yes i saw this PR, thanks !
